plugins/launchtube/src/pool.ts

```diff
@@ -0,0 +1,62 @@
+import * as fs from 'fs';
```

Comment:
interesting approach. Have you tried testing in docker env just in case there are no issuse or specific permissions are needed?

Reply:
I havent tried in docker but will test